lib/grizzly/zwave/commands/door_lock_operation_report.ex

```diff
@@ -167,8 +168,13 @@ defmodule Grizzly.ZWave.Commands.DoorLockOperationReport do
   # Version 4
   def decode_params(
         <<mode_byte, outside_handles_int::size(4), inside_handles_int::size(4),
-          door_condition_byte, timeout_minutes, timeout_seconds, target_mode_byte, duration_byte>>
+          door_condition_byte, timeout_minutes, timeout_seconds, target_mode_byte, duration_byte,
+          invalid_extra::binary>> = binary
       ) do
+    if byte_size(invalid_extra) != 0 do
+      Logger.warning("[Grizzly] Extra bytes in Door Lock Operation Report #{inspect(binary)}")
+    end
```

Comment:
I feel like the log message isn't needed since there's nothing that could be done about it and it doesn't seem particularly bad. Totally fine, though, if it's helpful for you.

Reply:
It's mostly because we don't really have a better way to monitor irregularities in incoming data right now. It's one of those things I'd rather have and not need than need and not have.